wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+---
+tags:
+  - UK Community Cup 7
+  - UKCC 7
+  - UKCC 7
+---
+
+# UK Community Cup 7
+
+![UKCC 7 logo](img/banner.png)
+
+The seventh edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.
+
+---
+
+## Introduction
+
+![Introduction](img/intro.png)
+
+- This is a **1v1, UK and Ireland based tournament**
```

Comment:
this feels more like a list of rules rather than an introduction. probably move to that section, and if you want any more of the important info here still you should use sentences/paragraphs instead of bullet points